CodeGeneration/Sources/SyntaxSupport/PatternNodes.swift

```diff
@@ -182,4 +182,23 @@ public let PATTERN_NODES: [Node] = [
     ]
   ),
 
+  Node(
+    kind: .editorPlaceholderPattern,
+    base: .pattern,
+    nameForDiagnostics: "editor placeholder",
+    documentation: """
+      An editor placeholder, e.g. `<#pattern#>` that is used in a position that expects a pattern.
+      """,
+    children: [
+      Child(
+        name: "placeholder",
+        deprecatedName: "identifier",
```

Comment:
This is a new node, so it doesn’t have to have a `deprecatedName`. Same for `EditorPlaceholderTypeSyntax`.

Reply:
🤦‍♂️